.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
     - PIP_CACHE_DIR="$HOME/.cache/pip"  # unify pip cache location for all platforms
 stages:
 #  - check
```

Comment:
minor: let's remove unused stages

Reply:
it might makes sense to setup stage with code formatting check as well